document/document_test.go

```diff
@@ -387,3 +387,19 @@ func TestGetTables(t *testing.T) {
 		t.Errorf("nested table not enumerated. found %d, expected 2", len(tables))
 	}
 }
+
+func TestExtractText(t *testing.T) {
+	testFilePath := "./testdata/extraction.docx"
+
+	want := "Line11 line12 line13.      Line14Line21Line31.    	Line32Line41Tab1Tab2Tab3"
```

Comment:
Do we not expect to get the linebreaks?  Is that stored in a different format?

Reply:
@gunnsth I'm not sure where to put the line breaks, this was one of the reasons I asked for a feedback. I may put it after each paragraph. Then the output will be like this:
```
Line11 line12 line13.      Line14
Line21
Line31.    
	Line32
Line41
Tab1
Tab2
Tab3
```